upgrade_tests/helpers/variants.py

```diff
@@ -7,71 +7,122 @@ class VersionError(Exception):
     """Error due to Unsupported Satellite Version"""
 
 
-supported_ver = ['6.1', '6.2']
-from_ver = os.environ.get('FROM_VERSION')
-to_ver = os.environ.get('TO_VERSION')
-
-if from_ver not in supported_ver:
-    raise VersionError(
-        'Unsupported preupgrade version {} provided for '
-        'entity variants existence tests'.format(from_ver))
-
-if to_ver not in supported_ver:
-    raise VersionError(
-        'Unsupported postupgrade version {} provided for '
-        'entity variants existence tests'.format(to_ver))
-
-
-# The dictionary of entity variants where the key is a component name
-# and value is list of all the component entity variants list those changes
-# during satellite versions
-#
-# Structure of variants directory:
-# {'component_name_e.g_filter,repository':
-#        [
-#        [variant1_6.1, variant1_6.2],
-#        [variant2_6.1, variant2_6.2],
-#        ]
-# }
-#
-# Note: The variants should be listed from 6.1 onwards
-entity_varients = {
-    'filter': [
-        ['lookupkey', 'variablelookupkey'],
-        ['(miscellaneous)', 'foremanopenscap::arfreport'],
-        ['organization', 'katello::subscription'],
-        ['configtemplate', 'provisioningtemplate'],
-        ['view_templates, create_templates, edit_templates, '
-         'destroy_templates, deploy_templates',
-         'view_provisioning_templates, create_provisioning_templates, '
-         'edit_provisioning_templates, destroy_provisioning_templates, '
-         'deploy_provisioning_templates']
-    ]
+_VERSION_NAMES_DIFF = {}
+_VERSION_NAMES_DIFF['6.1'] = {
```

Comment:
Instead of keeping 3 different directories for the versions that adds in unnecessary number of line of code.